Documents/UseCasesForWebsite.md

```diff
@@ -0,0 +1,39 @@
+Actors: Visitor, Artist, Admin, System 
+
+**Visitor** - A person which is visiting the website. 
+A visitor would use this website for getting information or donate as amputee a prosthetic or donate as a company. 
+Also the visitors can visit the gallery and like one of the displayed art there.
+A visitor 
+- **can request for a prosthetic**. Then the visitor needs to fill in a form, which can be reached by a link in the menu,  where the name, email-address and sort of prosthetic is filled in. 
+  When the visitor wants to pain the prosthetic then also the visitor can choose for a artist which the visitor likes or add some sketches of arts the visitor wants on the prosthetic.
+- **to donate a prosthetic, money or attention/advertisement** he/she fill in a form which can be found at the donate link in the menu of the site. The visitor needs to fill in a name, email address
+    maybe a telephone number and the visitor can choose by a select list what the visitor wants to donate. 
+- **goto the public gallery page** which can also be found with a link on the menu of the site. The images will be sorted with the latest one first. 
+- **View info page** by following the link on the menu of the website. On the info page the visitor can read information about the mission of this project and how it�s started, 
+   success stories and comments of the artist involved. 
+- **Like a picture**. When a visitor likes a picture, he/she can click on a like button. 
+- **View artwork**. When the visitor goes to the public gallery by clicking on the gallery link. When the visitor then clicks on a image shown in the gallery a modal screen is shown zoomed 
+  in at the whole screen where the image is also shown together with the name of the artist and the email address of the artist so the visitor can contact the artist who made that piece. 
+  Also  around 3 thumbnails of non-prosthetics) are shown , but also a reference to general other artwork
+
+**Artist** - The same as a visitor but he difference is that this person makes proffesinal art. 
+An artist could use this website for the same reasons as a regular user. Also the artist can fill in a form and 
+send two pictures of art the artist make so the admin can look f the artist is good enough to paint a prosthetic for a amputee or for the fun raising. 
+-  a artist **can apply to be a part of the project** by filling in a apply form with the name and email address , send in X images of work he/she made and gets approved or disapproved. 
```

Comment:
redundant. All the use cases for the artists have artist as a contextual actor- no need to mention it again. "can " is redundant.